nautobot/docs/development/getting-started.md

```diff
@@ -318,6 +318,21 @@ It may not always be convenient to enter into the virtual shell just to run prog
 $ poetry run mkdocs serve
 ```
 
+In order to serve docs on a Mac:
+
+* Install mkdocs via homebrew
```

Comment:
`poetry install` (as documented in the previous section) should install `mkdocs` as well as the other dependencies. You shouldn't need to globally install `mkdocs` on the system.